auto_nag/scripts/reminder.py

```diff
@@ -0,0 +1,79 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import datetime
+import re
+
+import dateutil.parser
+
+from auto_nag.bzcleaner import BzCleaner
+
+DATE_MATCH = re.compile(r"\[reminder-?[a-zA-Z0-9]* ([0-9]{4}-[0-9]{1,2}-[0-9]{1,2})\]")
```

Comment:
I can do this if you would prefer this; but my preference would be to keep the functionality able to be used generically for whatever people want. I think in an introduction email we can provide strong suggestions for what subtypes to specify ('compatfollowup', 'secapprovaltest', 'prefremoval') and that if we do want to improve the behavior for a subset of usage that will probably be driven by consumers telling us what they want...

Reply:
I prefer a limited scope so the functionality doesn't get abused. Also, this will give us more insights into the use cases (still people can request adding subtypes).

@marco-c wdyt?

Reply:
Okay; I've updated the patch.